atlasdb-ete-tests/Dockerfile

```diff
@@ -1,6 +1,23 @@
-FROM frolvlad/alpine-oraclejdk8:slim
-
-RUN apk update && apk add libstdc++ bash wget
+FROM centos:centos7
+
+MAINTAINER hsaraogi@palantir.com <Himangi Saraogi>
```

Comment:
didn't flag this the last time, but I think this should be `AtlasDB Team` (in case you move on to other projects)!